parcels/scripts/convert_npydir_to_netcdf.py

```diff
@@ -1,34 +1,46 @@
 from parcels import ParticleFile
 import numpy as np
 from os import path
+from glob import glob
 from argparse import ArgumentParser
 
 
-def convert_npydir_to_netcdf(tempwritedir):
+def convert_npydir_to_netcdf(tempwritedir_base, delete_tempfiles=False):
     """Convert npy files in tempwritedir to a NetCDF file
-    :param tempwritedir: directory where the temporary npy files are stored (can be obtained from ParticleFile.tempwritedir attribute)
+    :param tempwritedir_base: directory where the directories for temporary npy files
+            are stored (can be obtained from ParticleFile.tempwritedir_base attribute)
     """
 
+    tempwritedir = sorted(glob(path.join("%s" % tempwritedir_base, "*")),
+                          key=lambda x: int(path.basename(x)))[0]
     pyset_file = path.join(tempwritedir, 'pset_info.npy')
     if not path.isdir(tempwritedir):
         raise ValueError('Output directory "%s" does not exist' % tempwritedir)
     if not path.isfile(pyset_file):
         raise ValueError('Output directory "%s" does not contain a pset_info.npy file' % tempwritedir)
 
     pset_info = np.load(pyset_file, allow_pickle=True).item()
-    pfile = ParticleFile(None, None, pset_info=pset_info)
-    pfile.export()
-    pfile.dataset.close()
+    pfile = ParticleFile(None, None, pset_info=pset_info, tempwritedir=tempwritedir_base, convert_at_end=False)
+    if delete_tempfiles:
+        pfile.close()
```

Comment:
So `pfile.close()` does the full export, delete, and dataset.close()?
and if we don't want to delete tmp files, we only copy a few statements? Shouldn't delete_tempfiles be an argument of pfile.close()?